File: ServicioWeb/src/main/java/eitc/pucmm/servicioweb/classes/Item.java
package eitc.pucmm.servicioweb.classes;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Getter
@Setter
public class Item {
    @Id
    private Integer id;

    @Column(nullable = false)
    private ItemTag tag;

    private Integer count;

    public Item(Integer id, ItemTag tag, Integer count) {
        this.id = id;
        this.tag = tag;
        this.count = count;
    }

    public Item(){}
}

File: Midgarson/Assets/Scripts/GameCloud/Item.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class Item
{
    public long id;
    public ItemTag tag;
    public int count;

    public Item(long id, ItemTag tag, int count)
    {
        this.id = id;
        this.tag = tag;
        this.count = count;
    }
}
